libs/cardano-ledger-test/src/Test/Cardano/Ledger/STS.hs

```diff
@@ -328,45 +325,37 @@ prop_UTXOW =
 -- Test Tree
 ------------------------------------------------------------------------
 
-tests_STS :: TestTree
+tests_STS :: Spec
 tests_STS =
-  testGroup
-    "STS property tests"
-    [ govTests
-    , -- , utxoTests
-      -- TODO: this is probably one of the last things we want to
-      -- get passing as it depends on being able to generate a complete
-      -- `EpochState era`
-      testProperty "prop_EPOCH" prop_EPOCH
-      -- , testProperty "prop_LEDGER" prop_LEDGER
-    ]
-
-govTests :: TestTree
+  describe "STS property tests" $ do
+    govTests
+    -- utxoTests
```

Comment:
This module is messy - I know it's nothing to do with your changes, just wondering what we should do about it - tons of commented out tests and functions - unclear why they're commented out (I assume they're failing and/or not testing anything).
Realistically will we go back and fix them?   I'm inclined towards removing them  (in a future PR) if there is no clear plan to do something about them.

Reply:
I'm not sure about the commented-out stuff. This module seems to have had a lot of changes over the past few years.